airflow/jobs/job.py

```diff
@@ -17,6 +17,7 @@
 # under the License.
 from __future__ import annotations
 
+from datetime import datetime
```

Comment:
I think we have an effort consolidating this to `import datetime` instead.

Reply:
Ok. Done.
Unfortunately I can't move it under `TYPE_CHECKING` in pydantic/job.py  as Pydantic doesn't seem to like it.